packages/ses/demos/challenge/main.js

```diff
@@ -17,28 +17,20 @@
 //   limit attacker to some finite number of calls per go()
 //   framework updates UI (with setTimeout(0)), then calls go() again
 
-import('../../dist/ses.esm.js').then(({ lockdown }) => {
+lockdown();
+{
   console.log('starting');
 
   // Helpers
 
-  function $(selector) {
-    return document.querySelector(selector);
-  }
-  // function $$(selector) {
-  //   return document.querySelectorAll(selector);
-  // }
+  const $ = selector => document.querySelector(selector);
 
   // ********************
-  // 1. We build the SES Realm.
+  // 1. Should we endow the real `Date`?
   // ********************
 
-  const dateTaming = window.location.search.includes('dateNow=enabled')
-    ? 'unsafe'
-    : 'safe';
-  $('#dateNowStatus').textContent =
-    dateTaming === 'unsafe' ? 'Date.now() enabled' : 'Date.now() returns NaN';
```

Comment:
Why did these lines (setting of `#dateNowStatus`) get dropped?

Reply:
Baby bathwater exactly. My skipped past this. Thanks for catching!